docgen/src/guide/React_native.md

```diff
@@ -6,9 +6,643 @@ category: guide
 navWeight: 35
 ---
 
-React InstantSearch is compatible with [react-native](https://facebook.github.io/react-native/).
+React InstantSearch is compatible with [react-native](https://facebook.github.io/react-native/)
+
+In this tutorial, you’ll learn how to:
+
+* install react-instantsearch in your [React Native](https://facebook.github.io/react-native/)  project
+* bootstrap an app using the <InstantSearch> component
+* display infinite results from Algolia
+* add widgets to filter the results
+* connect your own component to the search
+
+## Before we start
+React InstantSearch is meant to be used with Algolia.
+
+Therefore, you’ll need the credentials to an Algolia index. To ease this getting started, here are the credentials to an already configured index:
+
+* appId: latency
+* searchKey: 3d9875e51fbd20c7754e65422f7ce5e1
+* indexName: ikea
+
+It contains sample data for an e-commerce website.
+
+This guide also expects you to have a working [React Native](https://facebook.github.io/react-native/)  project. If you need to setup a [React Native](https://facebook.github.io/react-native/)  project, we suggest you use [create-react-native-app](https://github.com/react-community/create-react-native-app) which is the [easiest way to start building a new React Native application](https://facebook.github.io/react-native/docs/getting-started.html). 
```

Comment:
a space to many after the React Native link

Also I don’t think it needs to be linked every time it’s mentioned